.circleci/config.yml

```diff
@@ -30,6 +30,50 @@ rc_branch_only: &rc_branch_only
       only:
         - /^Version-v(\d+)[.](\d+)[.](\d+)/
 
+rc_or_master_branch_only: &rc_or_master_branch_only
```

Comment:
It looks like this was accidentally restored. It's no longer being used.

Reply:
Whoops, okay removed it